src/HealthGPS/static_linear_model.cpp

```diff
@@ -85,6 +98,33 @@ void StaticLinearModel::linear_approximation(Person &person) {
     }
 }
 
+/// Initialises the weight of a person.
+///
+/// It uses the baseline adjustment to get its initial value, based on its sex and age.
+/// @param person The person fo initialise the weight for.
+/// @generator Random number generator for the simulation.
+void StaticLinearModel::initialise_weight(Person &person, Random generator) {
```

Comment:
Make `generator` a reference to avoid an unnecessary copy. If it can be const ref, that would be even better, but that might not be possible.